utils/string_utils.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+func IsNullOrEmpty(str string) bool {
+	return str == ""
```

Comment:
If the code shall catch all empty characters, including"    " cases - then it is better to have the one below.
s= strings.Trim(s, " ") == ""
return s==""

Reply:
Done